### DISCUSSION TITLE
/proxy returning 500

### DISCUSSION THREAD

**Discussion by kanhery** [grafana/grafana-infinity-datasource#126] [Q&A]
Hi there !

I ran the below :
_docker run -it -p 3000:3000 -e "GF_INSTALL_PLUGINS=yesoreyeram-infinity-datasource" grafana/grafana:latest_

Datasource addition works, but while trying to access an API, getting the below error --

Object
status:500
statusText:"Internal Server Error"
data:Object
error:"Internal Server Error"
response:"error getting response from http://localhost:8080/o/vf-cove-certmonitor/1.0/metrics
"
message:"error getting response from http://localhost:8080/o/vf-cove-certmonitor/1.0/metrics
"
config:Object
method:"POST"
url:"api/datasources/1/resources/proxy"
data:Object
retry:0
headers:Object
hideFromInspector:true
message:"error getting response from http://localhost:8080/o/vf-cove-certmonitor/1.0/metrics
"

Can you please help!

**Comment by yesoreyeram** [comment]
It looks like the URL is not accessible from the docker container.  See what you are getting curl for the URL from the grafana container. You may need to give correct URL of the API
